### DISCUSSION TITLE
BibTeX citation

### DISCUSSION THREAD

**Discussion by edsml-hmc122** [azavea/raster-vision#1788] [General]
Hi there, I am working on my MSc thesis in remote sensing and would like to mention this library in the related work section. Where can I find information on how to cite the this repository / its main authors?
Thank you

**Comment by AdeelH** [comment]
Hi, I have just added a `CITATIONS.cff` file to the repo, so you should be able to get the BibTeX from the repo main page:
![image](https://github.com/azavea/raster-vision/assets/13014700/b371f111-ae37-43fb-ad8c-2447c0f9c9fc)

Also, pasting here:
```BibTeX
@software{Azavea_Element_84_Raster_Vision_An,
author = {{Azavea/Element 84, Robert Cheetham}},
doi = {10.5281/zenodo.8018177},
title = {{Raster Vision: An open source library and framework for deep learning on satellite and aerial imagery (2017-2023).}},
url = {https://github.com/azavea/raster-vision}
}
```

**Reply by AdeelH** [reply]
@edsml-hmc122 I've updated the citation and my post above. Please use the updated citation.

**Comment by edsml-hmc122** [comment]
Perfect, thank you so much for the quick response!
